custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/impl/Builder.java

```diff
@@ -341,4 +350,43 @@ private void buildIfNeeded(@Nonnull DependencyInfo dep, @Nonnull String packagin
         mavenHelper.run(componentBuildDir, "versions:set", "-DnewVersion=" + newVersion);
         mavenHelper.run(componentBuildDir, combined);
     }
+
+    private static void unzip(File zipFile, File destDir) throws IOException {
```

Comment:
It is strange that there is no existing implementation in the dependencies

Reply:
Maybe I was not aware of it. Do you have anything in mind so that I can check?